### PR TITLE
issue-18093-my-account-form-broken

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: projects/dotcms-js/src/lib/core/core-web.service.ts [hunk 1]
```diff
@@ -11,7 +11,6 @@ import { Subject, Observable, throwError } from 'rxjs';
 import { map, catchError } from 'rxjs/operators';
 
 import {
-    hasContent,
```
Comment (fmontes) [inline projects/dotcms-js/src/lib/core/core-web.service.ts]: Does this have something to do with this PR?

File: src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.ts [hunk 2]
```diff
@@ -106,14 +106,18 @@ export class DotMyAccountComponent implements OnInit, OnDestroy {
                     }
                 };
 
-                this.form.valueChanges.pipe(takeUntil(this.destroy$)).subscribe(() => {
+                this.form.valueChanges.pipe(takeUntil(this.destroy$)).subscribe((valueChange) => {
```
Comment (fmontes) [inline src/app/view/components/dot-toolbar/components/dot-my-account/dot-my-account.component.ts]: type?

File: projects/dotcms-js/src/lib/core/core-web.service.ts (hunk 1 above)
Comment (alfredo-dotcms) [reply projects/dotcms-js/src/lib/core/core-web.service.ts]: not really, but it's causing a warning when running the project

**Comment by alfredo-dotcms** [comment]
@fmontes feedback uploaded

**Pull request merged**